flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolUtils.java

```diff
@@ -45,14 +50,49 @@
  */
 public class SlotPoolUtils {
 
+	public static final Time TIMEOUT = Time.seconds(10L);
+
 	private SlotPoolUtils() {
 		throw new UnsupportedOperationException("Cannot instantiate this class.");
 	}
 
+	static SlotPoolBuilder createSlotPoolBuilder(@Nullable final ResourceManagerGateway resourceManagerGateway) {
```

Comment:
This is not needed I think. It is used only in `SlotPoolImplTest` but that test class already has a `directMainThreadExecutor` defined.

Reply:
Removed.